modules/relax/ModuleConfig.cfc

```diff
@@ -155,9 +155,13 @@ component{
 			sessionsEnabled	= getApplicationMetadata().sessionManagement
 		};
 
-		// Apend it
+		// Append it
 		structAppend( configStruct.relax, relaxDSL, true );
 
+		// Set our server type for compatibility checks on java objects
```

Comment:
Yes, that's ok. Its the determining server type and server version. This is already done here: https://github.com/ColdBox/coldbox-platform/blob/development/system/core/util/CFMLEngine.cfc

Also, the `coldboxSettings` contain that info.  Dump them to check: `controller.getColdBoxSettings()`